app/models/post.rb

```diff
@@ -2,4 +2,14 @@ class Post < ApplicationRecord
   belongs_to :user
   has_many :likes
```

Comment:
- [ ] **[OPTIONAL]** Here too  `belongs_to :user` is supposed to be `belongs_to :author, class_name: 'User'`.